### PR TITLE
Updating display of application status on pet

### LINKED ISSUE #40: Withdrawn application should show "Application withdrawn" instead of "Application submitted"
https://jam.dev/c/3b8e38f6-8751-4025-83a6-d0e94749ce54

### PR DESCRIPTION
**Pull request by nsiwnf**
### Describe your change in plain English.
Updating display of application status on pet for adopter

**Staff view**
Screen Shot 2023-07-29 at 3 54 51 PM

**Adopter view**
https://github.com/rubyforgood/pet-rescue/assets/34173394/e78d81bb-25ec-4402-aac7-c53afc492d88

### Link to the issue
Resolves #40

### CODE REVIEW

**Comment by nsiwnf** [comment]
Only weird thing is that one of the statuses ends up being "Application Adoption Pending" - not sure if that's clear for the adopter?

**Review by Thrillberg — APPROVED** [review]
Woo hoo! Thank you!
